src/exprestive.coffee

```diff
@@ -80,20 +86,35 @@ class Exprestive
     @routesMethod @getRoutesHelperMethods()
 
 
+  # Save a function to @reversePaths to get a url back from a route name
+  registerReverseRoute: ({routeName, url}) ->
+    return unless routeName?
```

Comment:
Is silently ignoring a missing parameter like this the right thing? If this is an invalid situation, it shouldn't it throw an exception instead.